src/schemas/json/now.json

```diff
@@ -0,0 +1,183 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
Please make this `draft-04` since the editors interfacing with SchemaStore.org only supports that version.